sections/ads.txt

```diff
@@ -779,6 +779,7 @@ mno.hu##[id*="Banner"]
 mno.hu##[id*="ctnet"]
 mobilarena.hu##.adbox
 mobilarena.hu##[class*="adtext"]
+mobilarena.hu#?##page-forum #center > div:-abp-has(h4 + [onclick*="https://mobilarena.hu/muvelet/hozzaferes/kapcsolodas.php"])
```

Comment:
I think in this case you can merge domains, and this way, only 1 rule is enough:
```suggestion
mobilarena.hu,prohardver.hu#?##page-forum #center > div:-abp-has(h4 + [onclick*="https://mobilarena.hu/muvelet/hozzaferes/kapcsolodas.php"])
```

Reply:
The onclick target is different in the two rules.